### PR TITLE
ISSUE#64 - Login registration UI modified for mobile

### PR DESCRIPTION
**Pull request by Cookieth**
+ Added react native paper support
+ Added react native keyboard aware scroll view support
~ Modified input fields and margins
- Removed various unnecessary CSS parameters

NOTE: Temporary image templates have been added. Discuss with the UI team on what should actually be placed there.

### CODE REVIEW

File: mobile/src/package.json [hunk 1]
```diff
@@ -14,13 +14,15 @@
     "@react-navigation/native": "^5.7.3",
     "@react-navigation/stack": "^5.9.0",
     "axios": "^0.20.0",
-    "expo": "^38.0.0",
+    "expo": "^38.0.10",
+    "expo-cli": "^3.27.10",
```
Comment (deandrebaker) [inline mobile/src/package.json:18]: uhhhhhhhhhhhhh I think we should undo this. We aren't supposed to add expo-cli to the local packages. It should be global instead.

File: mobile/src/src/components/Login/LoginPage.jsx [hunk 2]
```diff
@@ -53,74 +61,108 @@ const LoginPage = ({ navigation }) => {
...
+          >
+            Register
+          </Button>
         </View>
-      </Modal>
-    </View>
+        <Modal visible={modalVisible} presentationStyle={"overFullScreen"}>
+          <View style={styles.container}>
+            <View style={styles.modalView}>
+              <Text style={{ alignItems: "center" }}>Error:</Text>
```
Comment (deandrebaker) [inline mobile/src/src/components/Login/LoginPage.jsx:115]: Let's add this style to the styles variable

**Review by deandrebaker — CHANGES_REQUESTED** [review]

**Review by trentstauff — DISMISSED** [review]
All looks good after you address Dre's requests.

Good work.

File: mobile/src/src/components/Login/LoginPage.jsx (hunk 2 above)
Comment (Cookieth) [reply mobile/src/src/components/Login/LoginPage.jsx]: not sure what you meant, let me know if the change I have just made is what you meant. I did not touch any of the modal info, this is all what it was before.

File: mobile/src/package.json (hunk 1 above)
Comment (Cookieth) [reply mobile/src/package.json]: ok

**Comment by Cookieth** [comment]
@deandrebaker ready for second review

**Review by deandrebaker — APPROVED** [review]
Good work!

**Pull request merged**